config.properties

```diff
@@ -0,0 +1,9 @@
+inference_address=http://127.0.0.1:8080
```

Comment:
@Arvind-Ideas2IT Can you please remove this file.

Reply:
Done.